cmd/pbm/list.go

```diff
@@ -322,7 +322,7 @@ func getSnapshotList(
 			PBMVersion:  b.PBMVersion,
 			Type:        b.Type,
 			SrcBackup:   b.SrcBackup,
-			StoreName:   b.Store.Name,
+			Profile:     b.Store.Name,
```

Comment:
@boris-ilijic  not 100% sure about this rename as it breaks backwards compatibility of the output. However we might want to unify on using "profile" instead of profile/storage (at least for user facing output)

Reply:
I know, this one is part of list/status output and the field is exposed only with json output format. That said, I think we should be consistent and use "profile" everywhere. Thus also changing it here. 

The actual definition for this one is of course https://github.com/percona/percona-backup-mongodb/pull/1232/files#diff-af738542639de6b3986d91c7462ac67697ce30184538554a7764f771a4c1a942R1156